cli-files/go-cat/cmd/root_test.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func TestRootCmd(t *testing.T) {
+	t.Run("No path provided", func(t *testing.T) {
+
+		cmd := rootCmd
+		err := cmd.Execute()
+
+		if err == nil {
+			t.Error("Expected error, got nil")
+		}
```

Comment:
It can be really useful to make assertions on the error message too, that way our tests show what happens when something goes wrong - this is particularly helpful at code review time, where the tests can answer questions like "Does the code give a _useful_ error message when this goes wrong?"

https://pkg.go.dev/github.com/stretchr/testify/require is a useful package for richer assertions in general, which may help out here.

Reply:
I have checked it out and seems like a useful package, thanks!